langserve/playground/src/components/ChatMessagesControlRenderer.tsx

```diff
@@ -101,6 +104,15 @@ function constructMessage(
   return null;
 }
 
+function isOpenAiFunctionCall(
+  x: unknown
+): x is { name: string; arguments: string } {
+  if (typeof x !== "object" || x == null) return false;
+  if (!("name" in x) || typeof x.name !== "string") return false;
+  if (!("arguments" in x) || typeof x.arguments !== "string") return false;
```

Comment:
Do we know behavior for an argumentless function?  empty dict encoded as json?

Reply:
Based on testing it should be an empty dict